docs/development/development-environment/macos/README.md

```diff
@@ -189,6 +194,20 @@ To configure the environment variables such as the number of web server threads
 the `.env.example` to `.env` and add the environment variables you want to configure. The variables will be
 automatically loaded to the application's environment.
 
+### Install other requirements
+
+Some operations in OpenProject require additional software. These include PDF exports and interaction with an
+LDAP server.
+
+In order to have OpenProject behave correctly, you need to install `imagemagick` and `java`.
```

Comment:
> In order to develop OpenProject in these areas, you need to install `imagemagick` and `java`.

Just a note, but OP behaves still correctly without the optional requirements, but of course without the features. 
As you explained in the chat, Java is only required if users want to develop something with LDAP, not running it for simple development.

Please ignore and resolve if it's too nitpicky.